src/main/java/org/prebid/server/bidder/beachfront/BeachfrontBidder.java

```diff
@@ -123,16 +121,33 @@ public BeachfrontBidder(String bannerEndpointUrl, String videoEndpointUrl) {
         final BeachfrontBannerRequest beachfrontBannerRequest = makeBannerRequest(bidRequest, bidRequest.getImp(),
                 errors);
         return Result.of(Collections.singletonList(
-                HttpRequest.<BeachfrontRequests>builder()
+                getHttpRequestBuilderWithDeviceHeaders(bidRequest.getDevice())
                         .method(HttpMethod.POST)
                         .uri(endpoint)
                         .body(Json.encode(beachfrontBannerRequest))
-                        .headers(BidderUtil.headers())
                         .payload(BeachfrontRequests.of(beachfrontBannerRequest, null))
                         .build()),
                 makeBadInputErrors(errors));
     }
 
+    //Also can include body as Object parameter, and set method POST by defoult
+    private static HttpRequestBuilder<BeachfrontRequests> getHttpRequestBuilderWithDeviceHeaders(Device device) {
+        final MultiMap headers = BidderUtil.headers();
+        if (device != null) {
+            addDeviceHeaders(headers, device);
+        }
+        return HttpRequest.<BeachfrontRequests>builder().headers(headers);
+    }
+
+    private static void addDeviceHeaders(MultiMap headers, Device device) {
+        HttpUtil.addHeaderIfValueIsNotEmpty(headers, HttpUtil.USER_AGENT_HEADER, device.getUa());
+        HttpUtil.addHeaderIfValueIsNotEmpty(headers, HttpUtil.X_FORWARDED_FOR_HEADER, device.getIp());
+        HttpUtil.addHeaderIfValueIsNotEmpty(headers, HttpUtil.ACCEPT_LANGUAGE_HEADER, device.getLanguage());
+
+        final Integer dnt = device.getDnt();
+        headers.add("DNT", dnt != null ? dnt.toString() : "0");
```

Comment:
Do we really need to set `DNT` header if `device.dnt` is null?
Pls see `org.prebid.server.bidder.gamoshi.GamoshiBidder#resolveHeaders` for reference.